vscode-extension/syntaxes/pico.tmLanguage.json

```diff
@@ -14,7 +14,15 @@
       "end": "(\\\")",
       "patterns": [
         {
-          "match": "((?:\\\")[\\x{01}-\\!\\#-\\x{10FFFF}]*?(?:\\\"))",
+          "match": "((?:\\\\)[\\\"\\\\])",
+          "captures": {
+            "1": {
+              "name": "string.quoted.double"
+            }
+          }
+        },
+        {
+          "match": "([\\x{01}-\\!\\#-\\[\\]-\\x{10FFFF}])",
```

Comment:
in the rascal grammar this is written as `![\\"]` right? can we not generate a negative regex class? `[^\\"]`?